src/react-components/ui-root.js

```diff
@@ -786,8 +798,8 @@ class UIRoot extends Component {
     const mediaStream = this.state.mediaStream;
 
     if (mediaStream) {
-      if (mediaStream.getAudioTracks().length > 0) {
-        console.log(`Using microphone: ${mediaStream.getAudioTracks()[0].label}`);
+      if (this.state.audioTrack) {
+        console.log(`Using microphone: ${this.state.audioTrack.label}`);
```

Comment:
Should we keep this console log since it was here before and is logging the "final state" of your microphone selection?

Reply:
Not sure if I get this one, the log line is still there, do you mean removing that log?